pkg/cmd/cmd.go

```diff
@@ -85,6 +85,8 @@ func NewCommand() *cobra.Command {
 	rootCmd.PersistentFlags().StringP("topic", "t", "", "Topic to submit branch to")
 	rootCmd.PersistentFlags().Bool("debug", false, "Run the command in debug mode")
 	rootCmd.PersistentFlags().String("remote", "", "Specifies the remote the review should be done on. By default the tracking remote of the target branch is used")
+	rootCmd.PersistentFlags().BoolP("work-in-progress", "w", false, "Mark the review as work in progress, or draft in compatible remotes. This flag is exclusively effective when creating Pull Requests")
+	rootCmd.PersistentFlags().BoolP("ready", "W", false, "Mark the review as ready in compatible remotes (i.e. removing the work in progress or draft flag)")
```

Comment:
It feels a bit strange to use the `W` flag for `ready`. what about `r` or `y`.

Reply:
Because this is how the original git-review exposes it